Signal/src/network/GiphyAPI.swift

```diff
@@ -105,20 +121,33 @@ enum GiphyFormat {
 
     public func pickStillRendition() -> GiphyRendition? {
```

Comment:
If we're going to use small animated previews, I wonder if even need still?

Reply:
Here's an example load (no scrolling)
```
2017-10-20 11:32:42:752 Signal[72855:2827693] [GiphyDownloader] Still bytes written:    000.06 MB
2017-10-20 11:32:42:752 Signal[72855:2827693] [GiphyDownloader] Animated bytes written: 001.10 MB
2017-10-20 11:32:42:752 Signal[72855:2827693] [GiphyDownloader] Total bytes written:    001.15 MB
```

S the stills are 20x smaller than even the "small" GIF renditions.

On fast connections, probably the stills are unnecessary, but on a slow connection, I think it makes sense to include them. Picking from a still then cancels any downloads of the animatedPreviews.